src/templates/component/TemplateComponent/index.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (C) 2012-2017  Online-Go.com
```

Comment:
geeze where did you dig that year from? Guess I missed the source for that a few times :sweat_smile:

Reply:
LOL yeah ... if you grep it you'll find plenty of hits, it turns out!  :o

Reply:
... yet another reason for having a template that is correct :D